### ISSUE TITLE
Lack of informations about Getting Started

### ISSUE THREAD

**Issue by Wykiki** [SFI-Mechatronics/FMI4j#31]
I'm currently trying to make FMI4j work on a sample project.

I encounter some Symbol resolution from examples provided in the [Wiki](https://github.com/SFI-Mechatronics/FMI4j/wiki/Co-Simulation).

My sample app :

```java
import no.mechatronics.sfi.fmi4j.common.FmiStatus;
import no.mechatronics.sfi.fmi4j.modeldescription.variables.RealVariable;

import java.io.File;

/**
 * Hello world!
 *
 */
public class App 
{
    public static void main( String[] args )
    {
        Fmu fmu = Fmu.from(new File("path/to/fmu.fmu")); // Fmu class not found
        try(FmiSimulation instance = fmu.asCoSimulationFmu().newInstance()) { // FmiSimulation class not found

            RealVariable myVar = instance.getVariableByName("myVar").asRealVariable()

            //assign custom start values
            myVar.setStart(2d);

            instance.init(); //throws on error

            double myValue = myVar.read().getValue(); //read
            FmiStatus status = myVar.write(5d); //write

            double dt = 1d/100;
            while (instance.getCurrentTime() < 10) {
                instance.doStep(dt);
            }

        } //instance is termi
    }
}
```

My pom.xml : 

```xml
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
  <modelVersion>4.0.0</modelVersion>

  <groupId>com.someone</groupId>
  <artifactId>javafmi</artifactId>
  <version>0.1</version>
  <packaging>jar</packaging>

  <name>javafmi</name>
  <url>http://maven.apache.org</url>

  <properties>
    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
  </properties>

  <dependencies>
    <dependency>
      <groupId>no.mechatronics.sfi.fmi4j</groupId>
      <artifactId>fmi-md</artifactId>
      <version>0.5.7</version>
    </dependency>

    <dependency>
      <groupId>junit</groupId>
      <artifactId>junit</artifactId>
      <version>3.8.1</version>
      <scope>test</scope>
    </dependency>
  </dependencies>
</project>
```

Have you any tips on how to make it work ?

Thanks in advance !

**Comment by Wykiki** [comment]
Hello, I was seeking for an FMI library on maven central (keyword 'fmi'), and fmi-md is the first I've found so I tried to make us of it.

When I read the Readme, I didn't see 'maven central' link on top of it, so I tried the artifact I previously saw on Maven Central, fmi-md:0.5.7.

Then I tried to make code living in the Wiki working, and obviously I missed that I didn't have to right artifact.

Now I've put the fmi-import:0.7.2 into my dependencies, but I can't build or run my main.
I don't have time to investigate more on this problem, but I could give you informations if you want to.

I'm currently using [javaFMI](https://bitbucket.org/siani/javafmi/wiki/FmuWrapper) to work with .fmu files.

And to finish with the 'Lack of informations', if you can, that would probably help other people if you put a pom.xml sample in this 'empty' [Wiki page](https://github.com/SFI-Mechatronics/FMI4j/wiki), or in the Getting started section.

Thank you for your time !

**Comment by markaren** [comment]
It would be nice to know what error you got with the fmi-import artifact.

Also I added some details to the main wiki page

**Comment by Wykiki** [comment]
I'm using Intellij IDEA, and I don't have logs to show the problem.

The error I've got is that i couldn't build or run my main class at the exact moment I added fmi-import artifact in the pom.xml dependencies field.

Just, the IDE wasn't even building the classes, even with obvious syntax problem in the code no build dialog was shown.

When I removed the artifact, I was able to build / run again.

But I guess it would be another thread to solve this problem !

Thanks for the Wiki update !
